var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -386,10 +380,6 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("adios2", when="+adios2")
     depends_on("blas")
     depends_on("boost+graph+math+exception+stacktrace", when="+boost")
-    # Need to revisit the requirement of STK
-    depends_on("boost+graph+math+exception+stacktrace", when="+stk")
```

Comment:
Also needs to be version dependent.

Reply:
Agreed. I will see if Alan can help me decide on a version.